app/assets/javascripts/snowcrash/modules/issues.js.coffee

```diff
@@ -5,7 +5,7 @@
 //= require snowcrash/modules/issues/tag-input
 
 document.addEventListener "turbolinks:load", ->
-  if $('body.issues').length
+  if $('body.issues, body.nodes, body.cards').length
```

Comment:
@MattBudz if we're making this a global feature, it needs to be extracted out of `issues.js`. It also needs to be based on data-behavior, so we don't have perpetuate this ugliness.